### PR TITLE
Fixed iteration 0 big decimal

### PR DESCRIPTION
**Pull request by deebot10**
We worked on our final issues on Iteration0. Fixing a problem with BigDecimal and adding conditionals that update the name, description, and unit price on the item repo.

### CODE REVIEW

File: lib/items_repo.rb [hunk 1]
```diff
@@ -80,13 +80,18 @@ def unit_price_exists?(attributes)
...
-      if unit_price_exists?(attributes)
-        item.unit_price = BigDecimal(attributes[:unit_price])
-      end
+    if item.nil?
+      nil
+      #attributes.each_key do |key|
+    elsif !attributes[:name].nil?
       item.name = attributes[:name]
+      item.updated_at = Time.now
+    elsif !attributes[:description].nil?
```
Comment (jrwhitmer) [inline lib/items_repo.rb:89]: I wonder if doing these elsifs will also prevent multiple attributes being updated at a time. Might need some helper methods here to reduce code length and make it easier to read.

File: spec/items_repo_spec.rb [hunk 2]
```diff
@@ -121,12 +121,12 @@
     it 'can update an item' do
       original_time = ir.find_by_id(263567475).updated_at
       attributes = {
-        unit_price: BigDecimal(379.99, 5),
```
Comment (jrwhitmer) [inline spec/items_repo_spec.rb:124]: Did we decide to stop using bigdecimal here because its already in our unit price attribute or for another reason?

**Review by jrwhitmer — APPROVED** [review]
This looks mostly good, I want to play around with the update method in items_repo when we next meet so that I can better understand what happens when we try to update multiple attributes at once.

File: spec/items_repo_spec.rb (hunk 2 above)
Comment (ahenzler) [reply spec/items_repo_spec.rb:124]: yes, it was already being called in the method.

**Review by ahenzler — APPROVED** [review]

**Review by rcasias — COMMENTED** [review]
Look great to me!

**Pull request merged**